ckanext/qgov/common/views/assets.py

```diff
@@ -0,0 +1,46 @@
+# encoding: utf-8
+
+from flask import Blueprint
+
+import re
+from ckan.plugins.toolkit import g, redirect_to, ObjectNotFound, render, h
+
+INCLUDED_BUNDLES = None
+
+blueprint = Blueprint(
+    u'asset_redirects',
+    __name__,
+    url_prefix=u'/assets/'
+)
+
+
+def init():
+    render(u'page.html')
+    global INCLUDED_BUNDLES
```

Comment:
This was defined in the module, so it's not necessarily to declare it here.

Reply:
I think it is needed here as the variable is changed within this function and flake8 complains that the local variable INCLUDED_BUNDLES is assigned but never used when I remove this, but the other global was fine to remove

Reply:
Ah. Looks like it might be best to drop the module level declaration on line 8, then, and just declare it as `global` in each relevant function.

Reply:
Or, you could create a single function that manages that variable. `any_redirect` would just call something like `get_included_bundles()`, which would do a null check, init if needed, and then return the list. No other function would need to know about the global variable.